src/Http/RequestFactory.php

```diff
@@ -78,7 +78,7 @@ private function getHttpHeaders(): array
 					continue;
 				}
 
-				$headers[strtr($key, '_', '-')] = $value;
+				$headers[str_replace('_', '-', $key)] = $value;
```

Comment:
Could you please provide a test for this change? Thanks 👍

Reply:
This call is equivalent but maintainable, I think that test is not necessary.
https://3v4l.org/MIT1e

Reply:
Okey